cicecore/cicedyn/analysis/ice_history.F90

```diff
@@ -1455,9 +1456,11 @@ subroutine init_hist (dt)
               ns1, f_FY)
 
       ! CMIP 2D variables (for "intensive" variables per Notz et al 2016 definition, 
-      ! use avg_ice_present = 'init', 'final', 'pond', or 'ridge' to divide by 
+      ! accumulate grid cell mean values as they are divided by the sum of ice fraction
+      ! Use avg_ice_present = 'init', 'final', 'pond', or 'ridge' to divide by 
       ! sum(aice), sum(apond), or sum(ardg) over time 
       ! aice is at the start of the timestep ('init') or the end of the timestep ('final')
+      ! avg_ice_present = 'none' means a standard time average
```

Comment:
Sorry, maybe too many holiday cookies are fuzzing up my brain, but I don't understand this description.  

In "accumulate grid cell mean values as they are divided by...", does "as" mean "because" or "simultaneously"?  Maybe take that line out altogether (as it was). 

Second, what is a "standard" time average?  The divisions by ice area are for spatial averaging, not time averaging. I suggest
> ! average_ice_present = 'none' does not divide by ice area

Reply:
The division by ice area is a time average when there is ice present.

Reply:
How does this sound?

      ! CMIP 2D variables (for "intensive" variables per Notz et al 2016 definition,
      ! that is a weighted time average when ice is present.
      ! Use avg_ice_present = 'init', 'final', 'pond', or 'ridge' to divide by
      ! sum(aice), sum(apond), or sum(ardg) over time
      ! aice is at the start of the timestep ('init') or the end of the timestep ('final')
      ! avg_ice_present = 'none' means a simple time average